core/preconditioner/jacobi.cpp

```diff
@@ -71,11 +72,14 @@ GKO_REGISTER_OPERATION(initialize_precisions, jacobi::initialize_precisions);
 template <typename ValueType, typename IndexType>
 void Jacobi<ValueType, IndexType>::apply_impl(const LinOp *b, LinOp *x) const
 {
-    using dense = matrix::Dense<ValueType>;
-    this->get_executor()->run(jacobi::make_simple_apply(
-        num_blocks_, parameters_.max_block_size, storage_scheme_,
-        parameters_.storage_optimization.block_wise, parameters_.block_pointers,
-        blocks_, as<dense>(b), as<dense>(x)));
+    precision_dispatch_spmv<ValueType>(
```

Comment:
Do we need the complex to real case here and would it/does it work?

Reply:
I don't see any reason why it shouldn't, since multiple RHS seem to work. It would probably be necessary for BJ-preconditioned IDR